test/client/main.cpp

```diff
@@ -281,9 +263,6 @@ int main(int argc, char* argv[])
     bool ensure_blob_class = false;
     bool download_first = false;
     // Client reset options are also used for async open.
-    std::string client_reset_metadata_dir = "";
```

Comment:
I still feel like I have no idea why all these seemingly unrelated changes around argument parsing and tcp delays are happening in this file.

Reply:
Do we even need this file? - this file is not used or compiled anymore, and it is so far out of date with the rest of the code that it will take a bit of effort to make current.

Reply:
I reverted the changes that were not related to updating the logger.